src/Tests/IQsharpEngineTests.cs

```diff
@@ -931,7 +931,7 @@ public async Task CompileAndSubmitWithClassicalControl()
                 .Input(source)
                     .ExecutesSuccessfully()
                 .WithMockAzure()
-                .Input("%azure.target honeywell.mock")
+                .Input("%azure.target quantinuum.mock")
```

Comment:
should we keep `honeywell` scenario here to make sure it continues to work?

Reply:
Fixed.